### DISCUSSION TITLE
Some similar apps

### DISCUSSION THREAD

**Discussion by DavidGP** [savbell/whisper-writer#67] [Ideas]
I just wanted to briefly point out two similar apps. There's [Whispering](https://github.com/braden-w/whispering) and [Whispertyping](https://whispertyping.com/#features). Both are excellent, but the latter stands out for its impressive extra features, such as the ability to respond to emails or rephrase selected text, with the convenience of performing these operations directly in the current window where the text resides.

Since I'm not sure whether Whispertyping will continue to be developed, I especially wanted to highlight this one. Anyone who wants to take a look at Whispertyping should check out the settings.json file, as it clearly shows how rather easily the above additional features can be implemented.

**Comment by SamirSaidani** [comment]
Thank you so much for letting us know! Unfortunately, it seems WhisperTyping isn't opensource, but the features are awesome.
